client/index.html

```diff
@@ -52,6 +52,26 @@
     </div>
     <div ng-include="'components/navbar/navbar.html'"></div>
 
+
+
+
```

Comment:
This would be great as a directive, where the appId comes from local.env.json locally, and an environment variable for CI and production.